tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+
+
+## Setting up
+
+In order to configure an environment for monitoring with the Particular Service Platform you will need to install and configure the components in the order listed.
+
+
+### Create Monitoring Instance
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system.
+
+Download and run the latest ServiceControl installer. This installer contains the ServiceControl Management Utility which can be used to create and manage Monitoring Instances. 
+
+SCREENSHOT - ServiceControl Installer
+
+TODO: For the pre-release version it is currently required to add a config file with a feature toggle to the SCMU. We should either document this step or remove it.
+
+Once the Management Utility is installed, run it.
+
+TODO: For the pre-release version it is currently required to install a valid license. We should either document this step or remove it.
+
+Click the **+ NEW INSTANCE** button and select **NEW MONITORING INSTANCE**.
+
+SCREENSHOT - SCMU Main Page with Drop-down open and NEW MONITOING INSTANCE selected
+
+Select a transport that the monitoring instance will use. This should be the same transport as your endpoints are running.
+
+NOTE: If you are using the sample solution, select the _Learning transport_ and in the _Connection String_ text box enter the folder that contains  the downloaded solution file.
+
+TODO: The Monitoring Instance currently cannot be configured to run with the Learning Transport. This feature needs to be implemented or the step above fixed.
+
+Click **Add** and the new monitoring instance will start.
+
+SCREENSHOT - SCMU Main Page with monitoring instance running
+
+
+### Install ServicePulse
+
+ServicePulse is a web application for production monitoring and recoverability. To enable it's monitoring features, you must connect ServicePulse to a ServiceControl Monitoring Instance.
+
+Download and run the latest ServicePulse installer. On the ServicePulse Configuration screen check the Monitoring option and enter the location of the monitoring instance API.
+
+SCREENSHOT - ServicePulse Setup - ServicePuse Configuration screen - Monitoring checked and filled in
+
+You can find the location of the monitoring instance API from within the ServiceControl Management Utility
+
+SCREENSHOT - SCMU - Monitoring Instance API highlighted
+
+Launch ServicePulse and navigate to the Monitoring tab.
+
+SCREENSHOT - ServicePulse Monitoring Tab - Empty
+
+NOTE: If Monitoring was not enabled during installation, the monitoring tab will not be visible.
+
+The final step is to configure some endpoints to send monitoring data.
+
+
+### Configure endpoints
+
+There are two NuGet packages to install into each endpoint. 
+
+TODO: Should the endpoints in the sample solution for this tutorial be already configured to send monitoring data OR should we explain how to do it?
```

Comment:
I would explain how to do it and leave comments in code as mentioned in https://github.com/Particular/docs.particular.net/pull/3299/files#r142863261